AutocompleteClient/FW/Logic/Worker/ConstructorIO.swift

```diff
@@ -27,9 +27,16 @@ public class ConstructorIO: AbstractConstructorDataSource {
     
     let clientID: String?
     
+    /**
+     Conversion property. It simplifies tracking conversions. To fully customize the data that's being sent, use ConstructorIO's CIOTracker protocol functions.
+     */
+    private(set) var conversion: CIOConversion!
+    
     public init(autocompleteKey: String, clientID: String?) {
         self.autocompleteKey = autocompleteKey
         self.clientID = clientID
+        
+        self.conversion = CIOConversion(tracker: self)
```

Comment:
I don't do swift, which means I don't understand how a customer will call the conversion function.  It looks like it might be implemented on a class they don't use directly?  Would it be possible to write include a conversion example of use in the sample app?  Like a random button somewhere?